tests/models/dpt/test_modeling_dpt.py

```diff
@@ -384,3 +389,30 @@ def test_post_processing_semantic_segmentation(self):
         segmentation = image_processor.post_process_semantic_segmentation(outputs=outputs)
         expected_shape = torch.Size((480, 480))
         self.assertEqual(segmentation[0].shape, expected_shape)
+
+    def test_post_processing_depth_estimation(self):
+        image_processor = DPTImageProcessor.from_pretrained("Intel/dpt-large")
+        model = DPTForDepthEstimation.from_pretrained("Intel/dpt-large")
+
+        image = prepare_img()
+        inputs = image_processor(images=image, return_tensors="pt")
+
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs)
+
+        predicted_depth = image_processor.post_process_depth_estimation(outputs=outputs)[0]
+        expected_shape = torch.Size((384, 384))
+        self.assertTrue(predicted_depth.shape == expected_shape)
+
+        predicted_depth_l = image_processor.post_process_depth_estimation(outputs=outputs, target_sizes=[(500, 500)])[
+            0
+        ]
```

Comment:
rather than splitting across three lines 

```suggestion
        predicted_depths = image_processor.post_process_depth_estimation(outputs=outputs, target_sizes=[(500, 500)])
        predicted_depth_l = predicted_depths[0]
```